src/fdk_fulltext_search/ingest/__init__.py

```diff
@@ -94,7 +96,7 @@ def fetch_all_content():
 
 
 def fetch_information_models():
-    info_url = f"{MODEL_HARVESTER_URI}/catalogs?catalogrecords=true"
+    info_url = f"{MODEL_HARVESTER_URI}/catalogs{RECORDS_PARAM}"
```

Comment:
Bruk gjerne params i requests.get() metode i stedet for direkte i url

Reply:
d04cd28

Reply:
Tror du også kan gjøre ca slik:
RECORDS_PARAM = {'catalogrecords':True} 
  
requests.get(url = info_url, params = RECORDS_PARAM)

Reply:
ba6957c